src/components/Dialogs/FittingDialog/FittingDialogComponent.scss

```diff
@@ -48,6 +48,32 @@
         margin-bottom: 0;
     }
 
+    .output-result-button {
+        position: absolute;
+        bottom: 25px;
+        right: 25px;
+        margin: 0 0 0 0;
+        transition: opacity 400ms;
+
+        .bp3-button {
+            transition: opacity 400ms;
+            user-select: none;
+        }
+    }
+
+    .output-log-button {
```

Comment:
Maybe we can combine `output-result-button` and `output-log-button` if their styles are the same.

Reply:
![image](https://user-images.githubusercontent.com/106953609/193226841-6be3ec84-f8db-4013-aebd-3e776f70a770.png)
They're combined into ".output-button"